usagov/vote-gov-drupal
# VOTE-2011: FAQ Schema

## Jira ticket

[VOTE-2011](https://cm-jira.usa.gov/browse/VOTE-2011)

## Description

Solves issues with schema markup by replacing double quotes with escaped double quotes and moves schemas into correct template so there is only one per page.

## Deployment and testing

### QA/Testing instructions

1. lando retune
2. go to state pages for each registration type
3. Open page source
4. Check that schema passes validator
5. Check that each page only has 1 schema

## Checklist for the Developer




## Checklist for the Peer Reviewers
